### DISCUSSION TITLE
Unable to highlight Solidity code in VsCode

### DISCUSSION THREAD

**Discussion by FrankezeCode** [Cyfrin/foundry-full-course-cu#1947] [General]
Please Fam,  I'm trying to write Solidity in VsCode , so I have downloaded the solidity extension, but yet the syntax highlighting in my code is not working properly, it can't differentiate variable type from variables name, everything is predominantly white.  

Please how can I resolve this issue.

The image is attached here.

![solidity image](https://github.com/Cyfrin/foundry-full-course-cu/assets/112127727/a736bdaf-375d-4fa4-b04d-165083c06ff3)

**Comment by EngrPips** [comment]
You should download any of the two extensions in the image below and make sure to enable them.

<img width="1440" alt="Screenshot 2024-07-05 at 06 37 43" src="https://github.com/Cyfrin/foundry-full-course-cu/assets/95760918/8fe9b7a5-d20b-40fc-816e-697de0cb45ca">

**Reply by FrankezeCode** [reply]
Thanks so much , 🙏 it's working now

**Reply by EngrPips** [reply]
Awesome, I am glad you are able to unblock yourself. Keep crushing it.
